### PR TITLE
Docs: Added note to contact Support

### PR DESCRIPTION
**Pull request by melgl**
This PR clarifies for Grafana Cloud customers to contact the Support team to configure enhanced LDAP.

### CODE REVIEW

File: docs/sources/setup-grafana/configure-security/configure-authentication/enhanced-ldap/index.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ weight: 900
 
 The enhanced LDAP integration adds additional functionality on top of the [LDAP integration]({{< relref "ldap/" >}}) available in the open source edition of Grafana.
 
-> **Note:** Available in [Grafana Enterprise]({{< relref "../../../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Advanced](/docs/grafana-cloud).
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Advanced](/docs/grafana-cloud). If you are a Grafana Cloud customer, please contact Support to configure this.
```
Comment (melgl) [inline docs/sources/setup-grafana/configure-security/configure-authentication/enhanced-ldap/index.md:21]:
```suggestion
> **Note:** Available in [Grafana Enterprise]({{< relref "../../../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Advanced](/docs/grafana-cloud). If you are a Grafana Cloud customer, please contact Support to configure this feature.
```

File: docs/sources/setup-grafana/configure-security/configure-authentication/enhanced-ldap/index.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ weight: 900
 
 The enhanced LDAP integration adds additional functionality on top of the [LDAP integration]({{< relref "ldap/" >}}) available in the open source edition of Grafana.
 
-> **Note:** Available in [Grafana Enterprise]({{< relref "../../../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Advanced](/docs/grafana-cloud).
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Advanced](/docs/grafana-cloud). If you are a Grafana Cloud customer, please contact Support to configure this feature.
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-security/configure-authentication/enhanced-ldap/index.md:21]: How do users contact support? Is there a form they complete, or do they need to call? If there's a form, we could add a link to it here.

Comment (melgl) [reply docs/sources/setup-grafana/configure-security/configure-authentication/enhanced-ldap/index.md:21]: Sorry for the delay @chri2547! Customers contact Support by opening a ticket from their Cloud Portal. We can use:

If you are a Grafana Cloud customer, please [open a support ticket in the Cloud Portal](/profile/org#support) to request this feature.

Comment (chri2547) [inline docs/sources/setup-grafana/configure-security/configure-authentication/enhanced-ldap/index.md:21]:
```suggestion
> **Note:** Available in [Grafana Enterprise]({{< relref "../../../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Advanced](/docs/grafana-cloud). If you are a Grafana Cloud customer, please [open a support ticket in the Cloud Portal](https://github.com/profile/org#support) to request this feature.

```

**Review by chri2547 — APPROVED** [review]
lgtm! Thanks for the contribution!!

File: docs/sources/setup-grafana/configure-security/configure-authentication/enhanced-ldap/index.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ weight: 900
 
 The enhanced LDAP integration adds additional functionality on top of the [LDAP integration]({{< relref "ldap/" >}}) available in the open source edition of Grafana.
 
-> **Note:** Available in [Grafana Enterprise]({{< relref "../../../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Advanced](/docs/grafana-cloud).
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Advanced](/docs/grafana-cloud). If you are a Grafana Cloud customer, please [open a support ticket in the Cloud Portal](https://github.com/profile/org#support) to request this feature.
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-security/configure-authentication/enhanced-ldap/index.md:21]:
```suggestion
> **Note:** Available in [Grafana Enterprise]({{< relref "../../../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Advanced](/docs/grafana-cloud). If you are a Grafana Cloud customer, please [open a support ticket in the Cloud Portal](https://grafana.com/orgs/$org/tickets) to request this feature.
```

**Pull request merged**